types/testing-library__react-hooks/index.d.ts

```diff
@@ -17,9 +18,16 @@ export interface HookResult<R> {
     readonly error: Error;
 }
 
+export interface WaitOptions {
+    timeout?: number;
+    suppressErrors?: boolean;
+}
+
 export interface RenderHookResult<P, R> {
     readonly result: HookResult<R>;
     readonly waitForNextUpdate: () => Promise<void>;
```

Comment:
`waitForNextUpdate` will also accept `WaitOptions` in `v3.2`

Reply:
Done.